doc/license/castcrafter/travel-anchors/license.md

```diff
@@ -0,0 +1,12 @@
+## Introduction 
+This file documents the allowed usage of code from the GitHub repo [TravelAnchors](https://github.com/castcrafter/travel_anchors).  
+As of today (07.10.21) all current code contributors(this doesn't include translators) noeppi_noeppi, CastCrafter, MelanX, cech12 and Robocraft999 have given permission to use their code for the following purposes:  
+- copy the code as a whole or in part
+- change the license of the copied and modified code to CC0, The Unlicense or any license we might see fit in the future
+
+## The permissions
+![CastCrafter](CastCrafter.png)
+![noeppi_noeppi](noeppi_noeppi.png)
+![MelanX](MelanX.png)
+![cech12](cech12.png)
+![RoboCraft999](RoboCraft999.png)
```

Comment:
broken link

Reply:
which one? they all highlight fine in my ide

Reply:
or only robocraft, rich diff doesn't work in PRs for images